frontend/.eslintrc

```diff
@@ -109,7 +106,9 @@
     "no-relative-import-paths/no-relative-import-paths": [
       "error",
       { "allowSameFolder": true, "rootDir": "src", "prefix": "src" }
-    ]
+    ],
+    // specifically for frontend/src/components/widgets/DataFrame/customCells
+    "import/no-named-as-default": 0
```

Comment:
Not sure how to resolve this because I think we need to `export default renderer` for the custom cell.